vzstorage-pd/virtuozzo-provisioner.go

```diff
@@ -162,12 +162,8 @@ func createPloop(mount string, options map[string]string) error {
 	}
 
 	// Create the ploop volume
-	cp := ploop.CreateParam{Size: volumeSize, File: ploopPath + "/" + options["volumeID"]}
-	if err := ploop.Create(&cp); err != nil {
-		return err
-	}
-
-	return nil
+	_, err = ploop.PloopVolumeCreate(ploopPath, volumeSize, "")
```

Comment:
I don't understand why the third argument is an empty string. I think we have to add a new parameter to a storage class which will say where delta-s have to be created.  We can call it as "deltaPath". If deltaPath isn't set we probably should create a regular ploop.